built-in-references/Kubernetes/container-probes-required/constraint.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sAzureContainerProbesRequired
+metadata:
+  name: container-probes-required
+spec:
+  match:
+    excludedNamespaces: {{ .Values.excludedNamespaces }}
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+  parameters:
+    probes : {{ .Values.requiredProbes }}
+    probeTypes: {{ .Values.supportedProbeTypes }}
```

Comment:
Please match rego policy and azure policy parameter names. We are looking towards auto-generating constraint files in the future.

Reply:
Adjusted the parameter names.